var/spack/repos/builtin/packages/getorganelle/package.py

```diff
@@ -6,14 +6,15 @@
 from spack.package import *
 
 
-class PyGetorganelle(PythonPackage):
+class Getorganelle(PythonPackage):
     """Organelle Genome Assembly Toolkit (Chloroplast/Mitocondrial/ITS)"""
 
     homepage = "https://github.com/Kinggerm/GetOrganelle"
     url = "https://github.com/Kinggerm/GetOrganelle/archive/refs/tags/1.7.5.0.tar.gz"
 
-    maintainers = ["dorton21"]
+    maintainers = ["snehring"]
```

Comment:
@dorton21 do you consent to no longer being the maintainer of this package? It's a list, so there's no reason you can't both maintain it.

Reply:
Desmond no longer works for iastate, and I suspect no longer has the time to contribute to the project. I'd just planned to assume responsibility for his packages as I touch them.

Reply:
Gotcha. It may be good to mass-update these in a single follow-up PR. I usually like to give maintainers up to a week to consent to being removed, and if they don't respond in a week well then they aren't maintaining anything anyway. Here is the full list:
```console
$ spack maintainers --by-user dorton21
gengeo        py-argh       py-gcs-oauth2-boto-plugin  py-google-apitools  py-lineenhancer  py-picrust2         py-rx              r-chipseq     r-fpcompare  r-qs             r-require        r-spades-tools
libdeflate    py-coveralls  py-getorganelle            py-gsutil           py-mrcfile       py-promise          py-sierrapy        r-circstats   r-memuse     r-quickplot      r-spades         r-stringfish
modeltest-ng  py-dadi       py-gffutils                py-haphpipe         py-mxfold2       py-responses        py-terminaltables  r-dexseq      r-pegas      r-rapiserialize  r-spades-addins  r-vcfr
py-ansi2html  py-flexmock   py-gooey                   py-iocapture        py-pathtools     py-retry-decorator  py-ucsf-pyem       r-fastdigest  r-pinfsc50   r-reproducible   r-spades-core    rnaquast
```